OxfordBodleian/Juel-Jensen/BDLaethd17.xml

```diff
@@ -33,87 +37,309 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                             <idno>Juel-Jensen 31</idno>
                         </altIdentifier>
                     </msIdentifier>
-                    
+
                     <msContents>
                         <summary/>
-                        
-                        <!--<msItem xml:id="ms_i1">
-                      
-                        </msItem>-->
-                        
+                        <msItem xml:id="ms_i1">
+                            <locus from="4ra" to="139ra"/>
+                            <title type="complete" ref="LIT2505Weddas" xml:lang="gez"/>
+                            <textLang mainLang="gez"/>
+
+
+                            <msItem xml:id="ms_i1.1">
+                                <locus from="4rb" to="19rb"/>
+                                <incipit xml:lang="gez">
```

Comment:
please add the locus element in incipit with its values corresponding to the extent of the incipit